provider/rancher/rancher_test.go

```diff
@@ -104,16 +103,19 @@ func TestRancherServiceFilter(t *testing.T) {
 		},
 	}
 
-	for _, e := range services {
-		actual := provider.serviceFilter(e.service)
-		if actual != e.expected {
-			t.Fatalf("expected %t, got %t", e.expected, actual)
-		}
+	for _, test := range testCases {
+		test := test
+		t.Run("", func(t *testing.T) {
```

Comment:
Can we use a non-empty first parameter for the test description?

(It's missing elsewhere too.)

Reply:
It's hard to named tests that I don't created. 😉

Reply:
How about using indices at least? It's far from ideal but better than nothing.

Reply:
Done!